src/Sylius/Bundle/CoreBundle/Distributor/TaxesDistributor.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Distributor;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class TaxesDistributor implements TaxesDistributorInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function distribute($taxItems, $taxAmount)
```

Comment:
As @pamil have written [here](https://github.com/Sylius/Sylius/pull/3868#issuecomment-173246708), I've fix it in https://github.com/Sylius/Sylius/pull/3907 (I mean arguments order ;))